__tests__/app.test.js

```diff
@@ -25,3 +25,23 @@ describe.only('GET /api/topics', () => {
 			})
 	})
 })
+describe('GET /api/articles', () => {
+	test('Should return a list of articles', () => {
+		return request(app).get('/api/articles')
+			.expect(200)
+			.then( (response) => {
+				expect(response.body.articles.length).toBe(12)	
+				response.body.articles.forEach( (article) => {
+					expect(article).toEqual(expect.objectContaining({
+						author: expect.any(String),
+						title: expect.any(String),
+						article_id: expect.any(Number),
+						topic: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number), 
+						comment_count: expect.any(String),
+					}))
+			})
+			})
+	})
+})
```

Comment:
Please write another test expecting a status 200 to test if the the articles are sorted by date in descending order. (jest-sorted).